models/yield_yak/yield_yak_withdraws.sql

```diff
@@ -0,0 +1,35 @@
+{{ config(
+	    schema = 'yield_yak',
+        alias = 'withdraws',
+        post_hook='{{ expose_spells(\'["arbitrum", "avalanche_c"]\',
+                                "project",
+                                "yield_yak",
+                                \'["angus_1"]\') }}'
```

Comment:
We may use the new format for all the `post_hook`:
```suggestion
	schema = 'yield_yak',
        alias = 'withdraws',
        post_hook='{{ expose_spells(
                      blockchains = \'["arbitrum", "avalanche_c"]\',
                      spell_type = "project",
                      spell_name = "yield_yak",
                      contributors = \'["angus_1"]\') }}'
```